src/govuk/components/header/header.js

```diff
@@ -1,46 +1,53 @@
-import '../../vendor/polyfills/Function/prototype/bind'
-import '../../vendor/polyfills/Event' // addEventListener and event.target normalization
+import '../../vendor/polyfills/Event'
 import '../../vendor/polyfills/Element/prototype/classList'
+import '../../vendor/polyfills/Function/prototype/bind'
 
 function Header ($module) {
   this.$module = $module
+  this.$menuButton = $module && $module.querySelector('.govuk-js-header-toggle')
+  this.$menu = this.$menuButton && document.getElementById(
```

Comment:
I've updated this to use `$module.querySelector( '#' + this.$menuButton.getAttribute('aria-controls'))` but I'd like to update this in 4.0 alongside #1809 and #1808?

I think generally where we're getting an element using an ID we should be able to assume that there will only be one element on the page with that ID, but agree that it could be a breaking change in some situations, where users currently have elements with duplicate IDs.

Reply:
👍 for the fix. As just discussed offline I'm not 100% sure about changing this to `document.getElementById` in 4.0 but we can discuss it down the line.